tests/components/zha/test_light.py

```diff
@@ -449,119 +474,180 @@ async def async_test_zha_group_light_entity(
     device_light_1._zha_gateway = zha_gateway
     device_light_2._zha_gateway = zha_gateway
     member_ieee_addresses = [device_light_1.ieee, device_light_2.ieee]
+    members = [GroupMember(device_light_1.ieee, 1), GroupMember(device_light_2.ieee, 1)]
+
+    assert coordinator.is_coordinator
 
     # test creating a group with 2 members
-    zha_group = await zha_gateway.async_create_zigpy_group(
-        "Test Group", member_ieee_addresses
-    )
+    zha_group = await zha_gateway.async_create_zigpy_group("Test Group", members)
     await hass.async_block_till_done()
 
     assert zha_group is not None
     assert len(zha_group.members) == 2
     for member in zha_group.members:
-        assert member.ieee in member_ieee_addresses
+        assert member.device.ieee in member_ieee_addresses
+        assert member.group == zha_group
+        assert member.endpoint is not None
 
-    entity_id = async_find_group_entity_id(hass, DOMAIN, zha_group)
-    assert hass.states.get(entity_id) is not None
+    device_1_entity_id = await find_entity_id(DOMAIN, device_light_1, hass)
+    device_2_entity_id = await find_entity_id(DOMAIN, device_light_2, hass)
+    device_3_entity_id = await find_entity_id(DOMAIN, device_light_3, hass)
+
+    assert (
+        device_1_entity_id != device_2_entity_id
+        and device_1_entity_id != device_3_entity_id
+    )
+    assert device_2_entity_id != device_3_entity_id
+
+    group_entity_id = async_find_group_entity_id(hass, DOMAIN, zha_group)
+    assert hass.states.get(group_entity_id) is not None
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5462445)